src/main/java/com/ita/if103java/ims/util/ItemEventUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.ita.if103java.ims.util;
+
+import com.ita.if103java.ims.dto.SavedItemDto;
+import com.ita.if103java.ims.entity.Event;
+import com.ita.if103java.ims.entity.EventName;
+import com.ita.if103java.ims.entity.User;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class ItemEventUtil {
+
+    public static Event createEvent(SavedItemDto savedItemDto, User user, EventName name) {
+        Event event = new Event();
+        event.setAccountId(user.getAccountId());
+        event.setDate(ZonedDateTime.now(ZoneId.systemDefault()));
+        event.setAuthorId(user.getId());
+        event.setWarehouseId(savedItemDto.getWarehouseId());
+        event.setName(name);
+        return event;
```

Comment:
please add message. For example:
"Add 52 Iphones from Supplier Z"
"Moved 52 Iphones  from Warehouse A"
"Moved 52 Iphones to Warehouse D"
"Sold 2 Iphones to Client C"